File: backend/admin-model/src/main/java/ru/ajana/admin/model/Role.java
package ru.ajana.admin.model;

import java.io.Serializable;

/**
 * Роль пользователя.
 *
 * @author Andrey Kharintsev by 10.03.2018
 */
public class Role implements Serializable {

  private Long id;
  private String name;
  private String ident;

  public Long getId() {
    return id;
  }

  public void setId(Long id) {
    this.id = id;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public String getIdent() {
    return ident;
  }

  public void setIdent(String ident) {
    this.ident = ident;
  }
}

File: client/app/model/Role.js
/**
 * Created by Andrey Kharintsev on 11.02.2019.
 */
Ext.define('Admin.model.Role', {
    extend: 'Admin.model.Base',

    fields: [
        { name: 'id',      type: 'int' },
        { name: 'name',     type: 'string' },
        { name: 'code',    type: 'string' }
    ]
});